wavpack-config.cmake.in

```diff
@@ -0,0 +1,10 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+if(NOT WIN32)
+  find_dependency(Threads)
+endif()
+find_dependency(Iconv)
```

Comment:
As far as I recall, iconv isn't needed in win32, and it is not needed for the library either.

Reply:
I don't know what the dependency is used for and these lines are copied from `find_package` in CMakeLists. If it is not needed, maybe CMakeLists.txt should be modified too

Reply:
> I don't know what the dependency is used for and these lines are copied from `find_package` in CMakeLists. If it is not needed, maybe CMakeLists.txt should be modified too

Cmake script justs checks for it, but it uses iconv only for programs when targeting non-windows. No change is really needed there.

Reply:
The main cmake script also does an unconditional `find_package(Iconv`) [here](https://github.com/dbry/WavPack/blob/511622eb3a98bd1c0cbef0c525b4d65dd3d89e77/CMakeLists.txt#L100-L103).
It should probably do this inside a condifitional `if(WIN32) ... endif()` block.

It looks like `Iconv::Iconv` is only used for the executable targets, not for the libraries.
So it can be removed here: the wavpack library does not have a dependency on it.
The executables might, but I think that's only a problem if you want to run these from within cmake.

Reply:
> The main cmake script also does an unconditional `find_package(Iconv`) [here](https://github.com/dbry/WavPack/blob/511622eb3a98bd1c0cbef0c525b4d65dd3d89e77/CMakeLists.txt#L100-L103). It should probably do this inside a condifitional `if(WIN32) ... endif()` block.
> 
> It looks like `Iconv::Iconv` is only used for the executable targets, not for the libraries. So it can be removed here: the wavpack library does not have a dependency on it. The executables might, but I think that's only a problem if you want to run these from within cmake.

So how should this part be changed? Would you please give a suggestion of change?

Reply:
Easy:
```suggestion
```